src/sidebar/app/components/Footer.js

```diff
@@ -287,11 +287,12 @@ class Footer extends React.Component {
               className="mdl-menu mdl-menu--top-right mdl-js-menu context-menu"
               data-mdl-for="context-menu-button">
               <li>
-                <button className="mdl-menu__item context-menu-item"
+                <a className="mdl-menu__item context-menu-item"
```

Comment:
Using <a> trigger some jsx-a11y errors on my machine

Footer.js: line 290, col 17, Error - Anchor used as a button. Anchors are primarily expected to navigate. Use the button element instead. (jsx-a11y/anchor-is-valid)
Footer.js: line 290, col 17, Error - Visible, non-interactive elements with click handlers must have at least one keyboard listener. (jsx-a11y/click-events-have-key-events)
Footer.js: line 290, col 17, Error - Static HTML elements with event handlers require a role. (jsx-a11y/no-static-element-interactions)

Reply:
Yes I just received some travis feedback about that.